bodhi/tests/server/consumers/test_updates.py

```diff
@@ -283,87 +269,75 @@ def test_update_not_found(self, work_on_bugs, fetch_test_cases, sleep):
         """
         If the message references an update that isn't found, assert that an Exception is raised.
         """
-        hub = mock.MagicMock()
-        hub.config = {'environment': 'environment',
-                      'topic_prefix': 'topic_prefix'}
-        h = updates.UpdatesHandler(hub)
+        h = updates.UpdatesHandler()
         h.db_factory = base.TransactionalSessionMaker(self.Session)
-        # Use a bogus alias to trigger the NotImplementedError.
-        message = {
-            'topic': 'bodhi.update.request.testing',
-            'body': {'msg': {'update': {'alias': 'does not exist'}}}}
-
+        # Use a bogus topic to trigger the NotImplementedError.
+        message = Message(
+            topic='bodhi.update.request.testing',
+            body={'msg': {'update': {'alias': 'does not exist'}}}
+        )
         with self.assertRaises(exceptions.BodhiException) as exc:
-            h.consume(message)
+            h(message)
 
         self.assertEqual(str(exc.exception), "Couldn't find alias 'does not exist' in DB")
         self.assertEqual(work_on_bugs.call_count, 0)
         self.assertEqual(fetch_test_cases.call_count, 0)
         sleep.assert_called_once_with(1)
 
+    @mock.patch('bodhi.server.consumers.updates.log.error')
+    @mock.patch('bodhi.server.consumers.updates.UpdatesHandler.fetch_test_cases')
+    @mock.patch('bodhi.server.consumers.updates.UpdatesHandler.work_on_bugs')
+    def test_without_alias(self, work_on_bugs, fetch_test_cases, error, sleep):
+        """
+        An error should get logged and the function should return if the message has no alias.
+        """
+        h = updates.UpdatesHandler()
+        h.db_factory = base.TransactionalSessionMaker(self.Session)
+        message = Message(
+            topic='bodhi.update.edit',
+            body={'msg': {'update': {}, 'new_bugs': ['12345']}}
+        )
+        h(message)
+
+>>>>>>> Replace fedmsg by fedora-messaging in consumers.
```

Comment:
Uh, this looks like a merge conflict leftover...

Reply:
Fixed